its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -205,6 +205,10 @@ public static List<Issues.Issue> getIssues(Orchestrator orch, String componentKe
     return newWsClient(orch).issues().search(new org.sonarqube.ws.client.issues.SearchRequest().setComponentKeys(Collections.singletonList(componentKey))).getIssuesList();
   }
 
+  public static List<Issues.Issue> getIssuesOnPR(Orchestrator orch, String componentKey, String pullRequestKey) {
```

Comment:
I would use `getIssues`. It has the pull request key as a parameter so it will be clear enough.